src/main/java/com/patreon/euphrates/Redshift.java

```diff
@@ -55,9 +55,8 @@ public void copyManifestPath(Config.Table table, String manifestPath) {
         String.format("COPY %s.%s\n", config.redshift.schema, Util.tempTable(table))
           + String.format("FROM 's3://%s/%s'\n", config.s3.bucket, manifestPath)
           + String.format(
-          "CREDENTIALS 'aws_access_key_id=%s;aws_secret_access_key=%s'\n",
```

Comment:
lets remove the accesskey and secretkey parts from the config too, and make iam role the only way to set things up if we're gonna go with this approach

Reply:
:+1: